README.md

```diff
@@ -127,7 +127,32 @@ All you need is Docker and git (optional).
 
    Note: You can navigate to the repo url and download the repo as a zip file if you don't want to use git
 
-2. Build the docker image: `$ docker build -t puppet-oneview .`
+2. You can build docker image locally or pull the docker image from Docker Hub.
+   #### Build docker image locally
+   * Build the docker image locally: `$ docker build -t puppet-oneview .`
+   #### Using Docker Hub Image
+   * Pull docker image from Docker Hub:
+   
+   The `hewlettpackardenterprise/hpe-oneview-sdk-for-puppet:<tag>`  docker image contains an installation of oneview-puppet installation you can use by just pulling down the Docker Image:
+
+   The Docker Store image tag consist of two sections: <sdk_version-OV_version>
+
+   ```bash
+   # Download and store a local copy of hpe-oneview-sdk-for-puppet and
+   # use it as a Docker image.
+   $ docker pull hewlettpackardenterprise/hpe-oneview-sdk-for-puppet:v2.6.0-OV5.2
+   # Run docker container with below commands.
+   $docker run -it --rm \
+     -v $(pwd)/:/puppet
+     -e ONEVIEW_URL='https://ov.example.com' \
+     -e ONEVIEW_USER='Administrator' \
+     -e ONEVIEW_PASSWORD='secret123' \
+     -e ONEVIEW_SSL_ENABLED=true, \
+     -e ONEVIEW_LOG_LEVEL='info' \
+     -e ONEVIEW_API_VERSION=800 \
+     hewlettpackardenterprise/hpe-oneview-sdk-for-puppet:v2.6.0-OV5.2 puppet apply fc_network.pp --debug --trace
+   ```
+
 
 3. Now you can run any of the example manifests in this directory:
```

Comment:
Remove 3 and please mention this as "Now you can run any of the example manifests in this directory locally"